quetz_frontend/src/components/Header.vue

```diff
@@ -14,8 +14,11 @@
         <img class="avatar-img" :src="avatar_url"  />
       </template>
       <template v-else>
-        <cv-button v-on:click="signin">
-          Sign In
+        <cv-button v-on:click="signinGithub">
```

Comment:
shoudl we have a config setting to enable / disable providers?

Reply:
I've just force pushed to make the google auth section optional, as well as only loading the router/module if configured.

I must admit I know very little Vue. Does it have a method to know if a given route exists (I see it does for vue routers, I don't know how that maps to fastapi/starlette though), maybe that could make these buttons dynamic with `v-if`?

I did think of doing a fetch on the route to see if it's 404's, but that sounds messy

Reply:
indeed an interesting question! Ideally this would be pre-rendered on the server side ... 
I'll merge this for now and think about this tomorrow.